Horatio/Operations/AlertOperation.swift

```diff
@@ -64,24 +64,39 @@ class AlertOperation: Operation {
     }
 
     override func execute() {
-        var presentationViewController: UIViewController?
-        
-        if let presentationContext = presentationContext {
-            presentationViewController = presentationContext
-        } else {
-            presentationViewController = UIApplication.shared.keyWindow?.rootViewController
+        // it's probably not safe to even walk the view hierarchy from a background thread, so do this all on main
+        DispatchQueue.main.async {
```

Comment:
With this new implementation, all of our interaction with the view hierarchy now happens in the main thread block. So there shouldn't be any problem with something else changing it "in between" any of our work. So I'd opt to leave it as async.